package.json

```diff
@@ -24,8 +24,8 @@
     "docker-build:dev": "docker build -f ./Dockerfile.dev -t iasql:latest-dev .",
     "docker-run": "docker run -p 9876:9876 -p 5432:5432 --name iasql iasql:latest",
     "docker-run:detach": "docker run -d -p 9876:9876 -p 5432:5432 --name iasql iasql:latest",
-    "docker-run:dev": "docker run -p 9876:9876 -p 5432:5432 -p 127.0.0.1:9229:9229 --name iasql-dev -v $PWD/src:/engine/src -v /engine/node_modules iasql:latest-dev",
-    "docker-run-vol": "docker run -p 9876:9876 -p 5432:5432 -v $(dirname $PWD)/db:/var/lib/postgresql/14/main --name iasql iasql:latest",
+    "docker-run:dev": "docker run -p 9876:9876 -p 5432:5432 -e IASQL_ENV=ci -p 127.0.0.1:9229:9229 --name iasql-dev -v $PWD/src:/engine/src -v /engine/node_modules iasql:latest-dev",
+    "docker-run-vol": "docker run -p 9876:9876 -p 5432:5432 -e IASQL_ENV=ci -v $(dirname $PWD)/db:/var/lib/postgresql/14/main --name iasql iasql:latest",
```

Comment:
why this? to not mess the telemetry?

Reply:
just read the comment above

Reply:
yes, the upgrade test is sending telemetry to posthog